README.md

```diff
@@ -2,3 +2,13 @@ Formbuilder
 ===========
 
 [![Build Status](https://travis-ci.org/Kinto/formbuilder.svg?branch=master)](https://travis-ci.org/Kinto/formbuilder)
+
+# Installation
+
+Want to test it out? To run the formbuilder locally, you can issue the
```

Comment:
Maybe invite them to test the UI on https://kinto.github.io/formbuilder/ first, then suggest a local installation next